pkg/proxy/proxy.go

```diff
@@ -24,6 +24,8 @@ var log = logger.Package()
 
 const (
 	ObotAccessTokenCookie      = "obot_access_token"
+	ObotAccessTokenCookieZero  = "obot_access_token_0"
+	ObotAccessTokenCookieOne   = "obot_access_token_1"
```

Comment:
Can this be made more generic so we can seamlessly handle an arbitrary number of cookies? I would hate for a random edge case to overflow the second cookie, too.

Reply:
This will change based on my other PR to refactor how we handle the cookies.